.github/workflows/build.yml

```diff
@@ -1,28 +1,80 @@
 name: 🏗️ Data Build
 
 on:
-  issues:
-    types: [opened]
+  workflow_dispatch:
+    inputs:
+      build_note:
+        description: "Note about this build"
+        type: string
+        required: false
+      logging_level:
+        description: "Logging level"
+        type: choice
+        required: true
+        options:
+          - WARNING
+          - INFO
+          - DEBUG
+  workflow_call:
+    inputs:
+      logging_level:
+        required: true
+        type: string
+env:
+  LOGGING_LEVEL: ${{ inputs.logging_level }}
+  BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
+  DO_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+  DO_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+  DO_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
 
 jobs:
   build:
-    if: >-
-      contains(github.event.issue.labels.*.name, 'data build')
+    name: Build Dataset
     runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgis/postgis:12-3.0-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
     steps:
       - uses: actions/checkout@v3
-      - name: config
+      - name: Logging check
         run: |
-          echo "running config"
+          python --version
+          echo "Hello logging level: ${{ inputs.logging_level }} (directly from action inputs)"
+          echo "Hello logging level: $LOGGING_LEVEL (from envar from action inputs)"
+          python -m python.run_logging
 
-      - name: data loading
+      - name: Setup python
+        uses: actions/setup-python@v4.5.0
+        with:
+          python-version: "3.11"
```

Comment:
yea it'd be nice to declare a version in one place (rather than in the 2 workflows and the dockerfile) but I didn't wanna go too far

and since the requirements are compiled when using the dev container, they'll be compatible with the version in the dockerfile